scripts/run_e2e_tests.sh

```diff
@@ -26,7 +26,7 @@ fi
 
 case "$target" in
   "test")
-    TAGS=""
+    TAGS="--include-tags cli"
```

Comment:
je recommande `local`
```suggestion
    TAGS="--include-tags local"
```

Reply:
Dac je modifie ça ✅